tests/cpp/utils.cpp

```diff
@@ -315,13 +315,17 @@ Container parse(const std::string& nvdisasm_output) {
   Container result;
   bool started = false;
   std::stringstream ss(nvdisasm_output);
-  std::string header;
-  std::regex find_header_regex(".text.(.+):");
+  std::regex zero_pattern_regex("/\\*0+\\*/");
```

Comment:
@zasdfgbnm Not sure why, but the pattern before this PR was like:

```
.text.ABC:
ABC:
/*0000*/ ...
...
```

With this PR, I'm seeing:


```
ABC:
.text.ABC:
/*0000*/ ...
...
```

The only plausible reason is I changed generated code to use namespace `nvf`, whereas it's currently anonymous. 

In either case, I just simplified the detection pattern to match with `/*0000*/`, which seems to be sufficient.